running-adapter.md

```diff
@@ -0,0 +1,55 @@
+# Running the REST FileSystem Adapter
+
+## Using Docker
+
+### Using Dropwizard Docker Image
+
+> :warning: Docker cannot be installed on Virtual Machines in our infrastructure.
```

Comment:
I'm not sure we should be talking about our infrastructure in an open-source project.